src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -45,16 +46,23 @@ public RawScriptResult(int? statusCode, object content)
             {
                 foreach (var header in Headers)
                 {
-                    response.Headers.Add(header.Key, header.Value?.ToString() ?? string.Empty);
+                    if (header.Key.Equals("content-type"))
+                    {
+                        response.ContentType = header.Value?.ToString();
```

Comment:
If the `content-type` is present and the value is `null` we should likely error here as the effect will be that the response content type will not be set. Throwing here with a clear message will save a fair amount of debugging time if this condition is found.